### PR TITLE
Add documentation about config and suppression

### PR DESCRIPTION
**Pull request by Corniel**
Documentation update required for https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/pull/124.

### CODE REVIEW

File: rules/configuration.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  
+  <PropertyGroup>
+    <NoWarn>Proj0010;Proj0011</NoWarn>
+  </PropertyGroup>
+  
+</PropertyGroup>
+```
+
+## Suppress specfic warnings
+Addopted from C-style languages it is possible to suppress
```
Comment (CptWesley) [inline rules/configuration.md:64]: Adopted

File: rules/configuration.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Configuration of .NET project file analyzers
+Although all **.NET project file analyzers** should work *as is* like a charm,
+there might be reasons to do some adjustments: disabling a specfic rule or
```
Comment (Sjaaky) [inline rules/configuration.md:3]: specific

File: rules/configuration.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+changing its serverity. The good news is: this can be done.
```
Comment (Sjaaky) [inline rules/configuration.md:4]: severity

File: rules/configuration.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+## Editor configuration
+Most (but not all) C# and VB.NET rules can be configured in the `.editorconfig`
+file. Unfortunatly, changing the severity (and other configuration) of rules 
+in the `.editorconfig` is [**NOT** supported by MS Build](https://github.com/dotnet/roslyn/issues/37876).
+
+## Analyzer INI file
+Fortunatly, it is possible to define project specfic preferences just as you
```
Comment (Sjaaky) [inline rules/configuration.md:12]: Fortunately

**Pull request merged**